server/package.json

```diff
@@ -18,7 +18,7 @@
     "@sentry/node": "^6.2.5",
     "@sentry/tracing": "^6.2.5",
     "axios": "^0.21.2",
-    "brave-alert-lib": "https://github.com/bravetechnologycoop/brave-alert-lib#v6.2.0",
+    "brave-alert-lib": "https://github.com/bravetechnologycoop/brave-alert-lib#td-update-dependencies",
```

Comment:
Reminder to self to change this once https://github.com/bravetechnologycoop/brave-alert-lib/pull/33 is merged.

Reply:
Done.